torch/csrc/jit/api/function_impl.h

```diff
@@ -36,9 +36,6 @@ struct TORCH_API GraphFunction : public Function {
 
   std::shared_ptr<Graph> optimized_graph() const override {
     std::lock_guard<std::recursive_mutex> lock(compile_mutex);
-    if (optimized_graph_) {
```

Comment:
This change here resolves the issue, since it allows the optimized_graph to be updated after it's set initially.

Reply:
cc @eellison